### ISSUE TITLE
typo in the docs

### ISSUE THREAD

**Issue by mattdiamond** [tapjs/tapjs#1043]
https://github.com/tapjs/tapjs/blob/ccfea67fec112bb14db260dcec3de02f83fdccb1/src/docs/_includes/why-tap.md#L39

**Comment by isaacs** [comment]
That's not a typo, they should be fun and helpful.

**Comment by mattdiamond** [comment]
I agree they should be fun and helpful, but they shouldn't fun and helpful. (The word "be" is missing.)
